app/services/fallback/__init__.py

```
@@ -0,0 +1,504 @@
+            await slack_alert.send(
+                title=f"⚠️ STT Failure on {provider} ({count}/{threshold})",
+                fields=[{"name": "Fail Count", "value": f"{count}/{threshold}"}],
+                sections=(
+                    [{"title": "Error", "text": f"```{error_msg}```"}]
+                    if error_msg
+                    else []
+                ),
+                fallback_text=f"STT failure on {provider} ({count}/{threshold})",
+                tag_users=_ALERT_TAG,
+            )
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)